### PR TITLE
Jv okta 299478 MFA Guide SMS update

### PR DESCRIPTION
**Pull request by jakubvul-okta**
## Description:
- **What's changed?** 

* Added SMS factor instructions to MFA Guide
* Renamed "Add Factor" section to "Enroll Factor"
* Fixed error in Factors Postman collection, updated JSON version as well as links
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-299478](https://oktainc.atlassian.net/browse/OKTA-299478)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/mfa/activate-factor/ga/activatefactor.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+1. Open Google Authenticator on your phone.
+2. Tap the plus sign **+** in the app.
+3. Select the **Scan barcode** option.
+4. Scan the QR Code on the browser tab that you opened previously.
+5. Switch to Postman and from the **Factor Lifecycle Operations** collection, select the **POST Activate TOTP Factor** request template.
+6. Replace the `{userId}` and `{factorId}` variables with the User ID and Factor ID values that you've copied previously.
+7. Select the **Body** tab and in the JSON body of the request, replace the `passCode` value with the passcode shown in the Google Authenticator app.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/activate-factor/ga/activatefactor.md:7]: Select the **Body** tab and, in the JSON body of the request, replace the.....

File: packages/@okta/vuepress-site/docs/guides/mfa/activate-factor/sms/activatefactor.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+1. Switch to Postman and from the **Factor Lifecycle Operations** folder, select the **Activate SMS Factor** request template.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/activate-factor/sms/activatefactor.md:1]: Switch to Postman and, in the **Factor Lifecycle Operations** folder, select the **Activate SMS Factor** request template.

File: packages/@okta/vuepress-site/docs/guides/mfa/activate-factor/sms/activatefactor.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+6. Replace the `{userId}` and `{factorId}` variables with the User ID and Factor ID values that you've copied previously.
+7. Select the **Body** tab and in the JSON body of the request, replace the `passCode` value with the passcode sent to you by SMS.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/activate-factor/sms/activatefactor.md:3]: Select the **Body** tab and, in the JSON body of the request, replace....

File: packages/@okta/vuepress-site/docs/guides/mfa/activate-factor/sms/activatefactor.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+8. Click **Send**.  A successful request results in an HTTP status code of `200` and a JSON payload response.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/activate-factor/sms/activatefactor.md:4]: extra space before the second sentence??

File: packages/@okta/vuepress-site/docs/guides/mfa/enroll-factor/sms/enrollfactor.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Using the ID for the User that you created earlier, add an SMS factor for that User:
+
+1. In Postman, open the **Factors (Okta API)** collection, and then the **Factor Lifecycle Operations** collection.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/enroll-factor/sms/enrollfactor.md:3]: no comma before and - only need a comma there if there is a separate action in the second clause.

File: packages/@okta/vuepress-site/docs/guides/mfa/enroll-factor/sms/enrollfactor.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+2. Select the **Enroll Okta SMS Factor** request template. The request appears on the right.
+3. In the request URL, replace the `{userId}` variable with the user ID that you saved in the <GuideLink link="../create-test-user">last step</GuideLink>.
+4. Replace the `{phoneNumber}` variable with the phone number you'd like to use for this test.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/enroll-factor/sms/enrollfactor.md:6]: ....with the phone number that you want to use for this test.

File: packages/@okta/vuepress-site/docs/guides/mfa/enroll-factor/sms/enrollfactor.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+4. Click **Send** to add the SMS Factor for your user. A successful request results in an HTTP status code of `200`, a JSON payload response, and an SMS stating "Your verification code is" followed by a numerical code. Save this code.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/enroll-factor/sms/enrollfactor.md:7]: A successful request results in an HTTP status code of `200` and a JSON payload response. The request results also include an SMS stating......

File: packages/@okta/vuepress-site/docs/guides/mfa/set-up-org/ga/enablemfa.md [hunk 8]
```diff
@@ -0,0 +1,3 @@
+1. From the Admin Console, select **Security** from the main menu and then select **Multifactor**.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/set-up-org/ga/enablemfa.md:1]: ....from the main menu, and then select **Multifactor**.

or

....from the main menu and then **Multifactor**.

File: packages/@okta/vuepress-site/docs/guides/mfa/set-up-org/sms/enablemfa.md [hunk 9]
```diff
@@ -0,0 +1,3 @@
+1. From the Admin Console, select **Security** from the main menu and then select **Multifactor**.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/set-up-org/sms/enablemfa.md:1]: ....from the main menu, and then select Multifactor.

or

....from the main menu and then Multifactor.

File: packages/@okta/vuepress-site/docs/guides/mfa/enroll-factor/sms/enrollfactor.md (hunk 7 above)
Comment (jakubvul-okta) [reply packages/@okta/vuepress-site/docs/guides/mfa/enroll-factor/sms/enrollfactor.md:7]: The results don't include an SMS, so I'll say "The request also results in an SMS".

File: packages/@okta/vuepress-site/docs/guides/mfa/enroll-factor/sms/enrollfactor.md [hunk 10]
```diff
@@ -0,0 +1,10 @@
+Using the ID for the User that you created earlier, add an SMS factor for that User:
+
+1. In Postman, open the **Factors (Okta API)** collection and then the **Factor Lifecycle Operations** collection.
+2. Select the **Enroll Okta SMS Factor** request template. The request appears on the right.
+3. In the request URL, replace the `{userId}` variable with the user ID that you saved in the <GuideLink link="../create-test-user">last step</GuideLink>.
+4. Replace the `{phoneNumber}` variable with the phone number you want to use for this test.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/enroll-factor/sms/enrollfactor.md:6]:
```suggestion
4. Replace the `{phoneNumber}` variable with the [phone number](/docs/reference/api/factors/#sms-profile) you want to use for this test.
```

File: packages/@okta/vuepress-site/docs/guides/mfa/enroll-factor/sms/enrollfactor.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (jakubvul-okta) [reply packages/@okta/vuepress-site/docs/guides/mfa/enroll-factor/sms/enrollfactor.md]: Link added.

**Pull request merged**